packages/repository-modernizer/src/create-base-project-structure.js

```diff
@@ -182,6 +182,22 @@ var CreateBaseProjectStructure = {
                     `CreateBaseProjectStructure: Base parent pom.xml created at ${projectPath}`
                 );
             }
+            if (projects.length > 1) {
+                fs.copyFileSync(
+                    path.join(basePath, constants.BASE_PARENT_POM),
```

Comment:
what if we have multiple parents  as we have in case of  National-instruments ,In this case we need to maintain variables accroding to parent of each projects 
<img width="404" alt="Screenshot 2021-05-31 at 12 31 39 PM" src="https://user-images.githubusercontent.com/30439863/120154089-1e152e00-c20d-11eb-84ba-1aa9b41a78b0.png">
@ManasMaji ,Should we consider this case ?
cc @dichaudhary  @ukgaurav